.github/workflows/ci.yml

```diff
@@ -78,6 +78,9 @@ jobs:
           rustc -Vv
           cargo -V
 
+      - name: Install hyperfine
+        run: cargo install hyperfine
```

Comment:
This looks like it will cost us a minute. :/ Are there binary releases we could install?

Also the Linux run in this PR took almost 1h which is a lot slower than what I remember... is that a coincidence or related to this PR?

Reply:
I'm not aware of binary distributions of hyperfine.

Perhaps we could install `dev` version, as we don't really care about measurements?
Though I'd expect hyperfine to be quick to build.

Reply:
It looks like Linux run is back to normal. Perhaps running `miri bench` only for host targets is a positive factor?

Reply:
It's about a minute to build. So not too bad I guess...

OTOH we don't actually care about the benchmarking part on CI, so we could as a hack also have CI create some `hyperfine` shell script that just executes its last argument. At least that would work on Linux. Whether it makes sense on Windows I cannot say.